5._Governance.md

```diff
@@ -6,57 +6,55 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
 
 ## 2. Decision Making.
 
 **2.1. Consensus-Based Decision Making.** The Project makes decisions through a consensus process ("Approval" or "Approved"). While the agreement of all applicable Participants is preferred, it is not required for consensus. Rather, the Maintainers or Steering Committee (as applicable) will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the applicable Project Participants and nature of support and objections. The Maintainers or Steering Committee (as applicable) will document evidence of consensus in accordance with these requirements.
 
-**2.2. Appeal Process.** Decisions may be appealed be via a pull request or an issue, and that appeal will be considered by the applicable Maintainers in good faith, who will respond in writing within a reasonable time.
+**2.2. Appeal Process.** Decisions may be appealed via a pull request or an issue, and that appeal will be considered by the applicable Maintainers in good faith, who will respond in writing within a reasonable time.
 
 **2.3. Steering Committee Appeal Process.** Decisions that have been appealed to the Maintainers may in extraordinary cases be appealed to the Steering Committee for reconsideration. An appeal to the Steering Committee must specify in detail (1) the specific decision that is being appealed; (2) the basis for contending that the decision was not aligned with the purposes, goals or scope of the Project; and (3) an explanation of why the decision is extraordinary enough to warrant an appeal to the Steering Committee. The appeal will be considered by the Steering Committee in good faith, who will respond in writing within a reasonable time. The Steering Committee may decline to consider appeals that are unexceptional, unfounded or excessive, including because of their repetitive character.
 
-**2.4. Amendments to Governance Documents.** The documents in this Governance repository may be amended by a two-thirds vote of the entire Steering Committee and are subject to approval by The Linux Foundation. However, entries may be added to the Notices file in this Governance repository as described therein.
+**2.4. Amendments to Governance Documents.** The documents in this Governance repository may be amended by four of the five Steering Committee Members and are subject to approval by The Linux Foundation. However, entries may be added to the Notices file in this Governance repository as described therein.
 
 ## 3. Ways of Working.
 
```

Comment:
I cant leave suggestions at the bottom of the doc, but here's a sixth section on GitHub layout. Maybe we put this in another doc rather than Governance, but I would like this somewhere to make sure we all agree on it going forward.

```
## 6. slsa-framework GitHub organization layout

* Team defined for the Steering Committee, e.g. slsa-sc-members. This team is given "admin" permissions over the organization and all of its repositories
* Each Workstream defines a team for Maintainers, e.g slsa-build-track-spec-maintainers, slsa-build-tooling-maintainers. Each team is given a "maintain" permission for the repository.
* For a repository with multiple Workstreams, a CODEOWNER file should include a line for each folder or file mapped to a Workstream team. The repository settings should mandate CODEOWNER approval before merge.
* A Workstream may define a GitHub team for code reviewers, e.g. slsa-build-tooling-reviewers. This team must contain only Contributors who have demonstrated knowledge of the codebase. This team is given "write" permission for a repository.
* The Steering Committee will create branch protection rules per repository for the "main" branch to prevent direct pushes. Additionally, the rules must mandate that only teams with "maintain" permissions can merge changes. This means that those with "write" can only review and approve changes, but those with "maintain" can merge as well.
* For all teams except the Steering Committee team, those added to the team should be a "member", not an "owner". "owners" can change who is in a team.
* Team memberships should be updated as Maintainers and Steering Committee member resign or are added
* All team membership updates should be transparently recorded on GitHub issues 
* The organization should only contain memberships for Contributors, Maintainers and Steering Committee members, and only if those members are in GitHub teams
```